src/main/java/org/zalando/putittorest/annotation/RestClient.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.putittorest.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@Target(ElementType.FIELD)
```

Comment:
I guess the whole thing only makes sense for field injection. That probably the reason why I don't feel like it's a pain to use `@Qualifier` because I never use field injection (except in tests), but exclusively constructor injection (to make it easier to test without Spring). In that case I only have to write `@Autowired` once.